feat: add `ndarray/base/rotr90` (#11030)
stdlib-js/stdlib: lib/node_modules/@stdlib/ndarray/base/rotr90/docs/repl.txt

{{alias}}( x, k, writable )
    Rotates a matrix (or a stack of matrices) 90 degrees clockwise.

    If `k > 0`, the function rotates the matrix clockwise. If `k < 0`, the
    function rotates the matrix counterclockwise.

    If provided an ndarray with fewer than two dimensions, the function returns
    a new view of the input ndarray.

    The returned ndarray is a *view* of the input ndarray. Accordingly, writing

>     to the original ndarray will mutate the returned ndarray and vice versa.

Reviewer: ```suggestion
    The returned ndarray is always a *view* of the input ndarray. Accordingly,
    writing to the original ndarray will mutate the returned ndarray and vice
    versa.

    If provided an ndarray with fewer than two dimensions, the function does not
    perform a rotation and simply returns a new view of the input ndarray.
```

    The `writable` parameter only applies to ndarray constructors supporting

>     read-only instances.

Reviewer: ```suggestion
```

    Parameters
    ----------
    x: ndarray
        Input array.

    k: integer
        Number of times to rotate by 90 degrees.

    writable: boolean

>         Boolean indicating whether the returned ndarray should be writable.

Reviewer: ```suggestion
        Boolean indicating whether a returned ndarray should be writable. This
        parameter only applies to ndarray constructors which support read-only
        instances.
```

    Returns
    -------
    out: ndarray

>         Output array.

Reviewer: ```suggestion
        Output array view.
```

    Examples
    --------
    > var x = {{alias:@stdlib/ndarray/array}}( [ [ 1, 2 ], [ 3, 4 ] ] )
    <ndarray>[ [ 1, 2 ], [ 3, 4 ] ]
    > var y = {{alias}}( x, 1, false )
    <ndarray>[ [ 3, 1 ], [ 4, 2 ] ]
    > y = {{alias}}( x, 2, false )
    <ndarray>[ [ 4, 3 ], [ 2, 1 ] ]
    > y = {{alias}}( x, 3, false )
    <ndarray>[ [ 2, 4 ], [ 1, 3 ] ]

> 

Reviewer: ```suggestion
```
